styles.css

```diff
@@ -0,0 +1,736 @@
+}
+
+.card-logo {
+  width: 20%;
+  padding: 4%;
+}
+
+.card-subject {
+  width: 30%;
+  font-family: Lato, sans-serif;
+  font-weight: bolder;
+}
+
+.card-description {
+  width: 50%;
+  padding: 2% 2% 2% 0%;
+}
+
+.content-join {
+  align-self: center;
+  background-color: #ec5242;
+  color: #f8f8f8;
+  font-size: 16px;
+  width: 60%;
+  height: 70px;
+  margin-top: 5%;
+  border-color: #ec5242;
+}
+
+.content-program-dsk {
+  display: none;
+}
+
+.featured {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 5% 2%;
+}
+
+.featured-title {
+  font-family: Lato, sans-serif;
+  text-transform: none;
+  color: #313336;
+  font-size: 24px;
+  font-weight: bold;
+  margin-bottom: 10%;
+}
+
+.featured-card {
+  display: flex;
+}
+
+.card-dsk {
+  display: none;
+}
+
+.card-left {
+  width: auto;
+  margin: 2%;
+}
+
+.card-right {
+  width: auto;
+  margin: 2%;
+}
+
+.right-song-name {
+  color: #272a31;
+  font-family: Lato, sans-serif;
+  font-weight: bolder;
+}
+
+.right-performance {
+  color: #ec5242;
+  font-style: italic;
+  margin-bottom: 5%;
+}
+
+.right-summary {
+  color: #272a31;
+  font-size: 16px;
+}
+
+.featured-button {
+  width: 100%;
+  height: 50px;
+  margin: 10% 1% 5%;
+  background-color: white;
+  border: solid #d3d3d3;
+  color: #272a31;
+}
+
+.main-partners {
+  display: none;
+}
+
+.main-footer {
+  display: none;
+}
+
+/* About Page */
+
+.about-main {
+  display: flex;
+  flex-direction: column;
+  padding: 2% 6% 8%;
+  background-color: #d3d3d34d;
+  background-image: url("./images/main_bg.jpg");
+  background-size: cover;
+}
+
+.about-hello {
+  color: #ec5242;
+  font-size: 24px;
+  margin-bottom: 1%;
+  text-align: center;
+}
+
+.about-h1 {
+  margin-bottom: 5%;
+  text-align: center;
+}
+
+.about-description {
+  text-align: center;
+  color: #272a31;
+  background-color: white;
+  padding: 5% 2%;
+  line-height: 24px;
+}
+
+.about-contact {
+  color: #272a31;
+  text-align: center;
+  margin: 10% 10% 0%;
+}
+
+.about-address {
+  text-align: center;
+}
+
+.about-email-a {
+  color: #272a31;
+  font-style: normal;
+}
+
+.about-logo {
+  display: flex;
+  flex-direction: column;
+  padding: 8% 6%;
+  border-bottom: solid #272a314b;
+}
+
+.about-h2 {
+  font-family: Lato, sans-serif;
+  text-transform: capitalize;
+  text-align: center;
+  color: #272a31cc;
+  font-size: 20px;
+  font-weight: bold;
+  margin-bottom: 2%;
+}
+
+.logo-desc {
+  color: #272a31;
+  text-align: center;
+  margin: 2%;
+}
+
+.about-logo-img {
+  align-self: center;
+  max-width: 80vw;
+  border: #272a313a;
+  border-style: solid;
+  margin-top: 10%;
+}
+
+.about-past {
+  display: flex;
+  flex-direction: column;
+  padding: 8% 6%;
+}
+
+.past-title {
+  font-family: Lato, sans-serif;
+  text-align: center;
+  color: #272a31e0;
+  font-weight: bold;
+  text-transform: none;
+}
+
+.past-description {
+  color: #272a31;
+  text-align: center;
+  margin: 5% 0%;
+}
+
+.past-imgs {
+  margin: 2% 0%;
+}
```

Comment:
- [ ] Right now, the past image is overflowing. However, I fixed this overflowing by giving this class a `width: 100%`. See the image below.

Your Implementation | Cindy's Design
:---:|:---:
![](https://i.imgur.com/5U8ZgDa.png) | ![](https://i.imgur.com/qd5s2fT.png)